cmake/Corrosion.cmake

```diff
@@ -1308,19 +1308,21 @@ function(corrosion_install)
 
                     if(EXPORT_NAME)
                         get_target_property(COR_FILE_NAME ${INSTALL_TARGET}-static COR_FILE_NAME)
-                        file(APPEND
-                            ${CMAKE_BINARY_DIR}/corrosion/${EXTRA_TARGETS_EXPORT_NAME}
+                        if(NOT EXISTS ${CMAKE_CURRENT_BINARY_DIR}/corrosion/${EXTRA_TARGETS_EXPORT_NAME}-static-exists)
```

Comment:
Can we assume that `corrosion_install()` will only be called once per target? If yes perhaps we could delete the file first.
We would still want to overwrite the file in case `$DESTINATION`  changed.

Reply:
I guess we could, but the file is at the EXPORT level and we don't know if other installs put something else in there.

I'm not entirely sure what the completely correct solution is here.

Reply:
I pushed a suggestion - If we require that the export name is unique, then we shouldn't have any problems with other `corrosion_installs` putting information into the same export.
What do you think?

Reply:
I think that makes it good enough. The worst that happens is that people need to add another line to their config file.